src/pykeen/evaluation/classification_evaluator.py

```diff
@@ -33,9 +33,14 @@ class ClassificationMetricResults(MetricResults):
     @classmethod
     def from_scores(cls, y_true, y_score):
         """Return an instance of these metrics from a given set of true and scores."""
-        return ClassificationMetricResults(
-            {key: metric.score(y_true, y_score) for key, metric in CLASSIFICATION_METRICS.items()}
-        )
+        data = dict()
+        for key, metric in CLASSIFICATION_METRICS.items():
+            value = metric.score(y_true, y_score)
+            if not isinstance(value, (int, float)):  # numpy.float
+                assert hasattr(value, "item") and callable(value.item)
+                value = value.item()
```

Comment:
maybe make this a bit more specific to send errors for unexpected datatypes?

Reply:
c984ebd

I did not emit a warning, since this might be raised quite often (e.g., for using classification based metrics in evaluation callbacks / early stopping).